packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx

```diff
@@ -60,7 +60,7 @@ const SubscriptionWarningModal: React.FC = () => {
       handleClose={() => {
         return
       }}
-      title="Automatisk avtalskontroll sker för samtliga användare"
```

Comment:
This text should come from the resource link

Reply:
Since the title and text in the modal doesn't have any business logic we decided that it was going to stay in the frontend code to make it easier to maintain.